src/labone/core/hpk_schema.py

```diff
@@ -1021,5 +517,1153 @@ def get_schema_loader() -> zhinst.comms.SchemaLoader:
+
+    def __init__(self, **kwargs):
+        super().__init__(get_schema_loader(), self.capnp_id, **kwargs)
+
+
+class SessionListNodesResults(zhinst.comms.DynamicStruct):
+    capnp_id = 0xEBD34BDB74C961B0
+    paths: list[str]
+
+    def __init__(self, **kwargs):
+        super().__init__(get_schema_loader(), self.capnp_id, **kwargs)
+
+    def init_paths(self, size: int) -> list[str]:  # type: ignore[valid-type]
+        return super().init_list("paths", size)  # type: ignore[return-value]
+
+
+class SessionDeprecatedGetValuesParams(zhinst.comms.DynamicStruct):
+    capnp_id = 0xCD242E1BB990DFE7
+    paths: list[str]
+    client: bytes
+
+    def __init__(self, **kwargs):
+        super().__init__(get_schema_loader(), self.capnp_id, **kwargs)
+
+    def init_paths(self, size: int) -> list[str]:  # type: ignore[valid-type]
+        return super().init_list("paths", size)  # type: ignore[return-value]
+
+
+class Result(zhinst.comms.DynamicStruct):
+    capnp_id = 0xBAB0F33E1934323D
+    ok: t.Any
+    err: t.Any
+
+    def __init__(self, **kwargs):
+        super().__init__(get_schema_loader(), self.capnp_id, **kwargs)
+
+
+class AnnotatedValue(zhinst.comms.DynamicStruct):
+    capnp_id = 0xF408EE376E837CDC
+    metadata: AnnotatedValueMetadata
+    value: Value
+
+    def __init__(self, **kwargs):
+        super().__init__(get_schema_loader(), self.capnp_id, **kwargs)
+
+    def init_metadata(self, **kwarg) -> AnnotatedValueMetadata:  # type: ignore[valid-type]
+        return super().init_struct("metadata", **kwarg)  # type: ignore[return-value]
+
+    def init_value(self, **kwarg) -> Value:  # type: ignore[valid-type]
+        return super().init_struct("value", **kwarg)  # type: ignore[return-value]
+
+
+class Error(zhinst.comms.DynamicStruct):
+    capnp_id = 0xC4E34E4C517D11D9
+    code: int  # uint32
+    message: str
+    category: str
+    kind: ErrorKind
+    source: str
+
+    def __init__(self, **kwargs):
+        super().__init__(get_schema_loader(), self.capnp_id, **kwargs)
+
+
+class SessionDeprecatedGetValuesResults(zhinst.comms.DynamicStruct):
+    capnp_id = 0xE948518C129575BB
+    result: list[Result]
+
+    def __init__(self, **kwargs):
+        super().__init__(get_schema_loader(), self.capnp_id, **kwargs)
+
+    def init_result(self, size: int) -> list[Result]:  # type: ignore[valid-type]
+        return super().init_list("result", size)  # type: ignore[return-value]
+
+
+class Value(zhinst.comms.DynamicStruct):
+    capnp_id = 0xB1838B4771BE75AC
+    int64: int  # uint64
+    double: float  # float64
+    complex: complex
+    string: str
+    vectorData: VectorData
+    cntSample: CntSample
+    triggerSample: TriggerSample
+    none: VoidStruct
+    streamingError: Error
+    shfDemodData: ShfDemodulatorVectorData
+    shfResultLoggerData: ShfResultLoggerVectorData
+    vectorCntSamples: list[CntSample]
+    vectorTriggerSamples: list[TriggerSample]
+    largeVectorData: LargeVectorData
+    bool: bool
```

Comment:
Would it make sense to move `bool` above the struct?

So the ordering of init functions would match the attribute order

Reply:
Yes, from that perspective, it makes sense. But this struct is autogenerated match the capnp struct, which looks as the following:

```
struct Value @0xb1838b4771be75ac {
  union {
    int64                    @0  :Int64;
    double                   @1  :Float64;
    complex                  @2  :Complex;
    string                   @3  :Text;
    vectorData               @4  :VectorData;
    cntSample                @5  :CntSample;
    triggerSample            @6  :TriggerSample;
    none                     @7  :VoidStruct;
    streamingError           @8  :StreamingError;
    shfDemodData             @9  :ShfDemodulatorVectorData;
    shfResultLoggerData      @10 :ShfResultLoggerVectorData;
    vectorCntSamples         @11 :List(CntSample);
    vectorTriggerSamples     @12 :List(TriggerSample);
    largeVectorData          @13 :LargeVectorData;
    bool                     @14 :Bool;
    shfScopeData             @15 :ShfScopeVectorData;
    shfGeneratorWaveformData @16 :ShfGeneratorWaveformVectorData;
    shfPidData               @17 :ShfPidVectorData;
  }
}
```

Reply:
That is what I figured, thanks